plugins/InspectionGadgets/src/META-INF/InspectionGadgets.xml

```diff
@@ -2690,6 +2690,11 @@
                      groupKey="group.names.verbose.or.redundant.code.constructs" enabledByDefault="true" level="WARNING"
                      editorAttributes="NOT_USED_ELEMENT_ATTRIBUTES"
                      implementationClass="com.siyeh.ig.controlflow.UnnecessaryReturnInspection" cleanupTool="true"/>
+    <localInspection groupPath="Java" language="JAVA" suppressId="RedundantCompareToJavaTimeStatement" shortName="RedundantCompareToJavaTime"
+                     groupBundle="messages.InspectionsBundle" groupKey="group.names.verbose.or.redundant.code.constructs"
+                     enabledByDefault="false" level="WARNING" editorAttributes="NOT_USED_ELEMENT_ATTRIBUTES"
```

Comment:
1. I think this inspection is okay to enable by default. 
2. Please remove the `editorAttributes` attribute, because not-used highlighting is reserved for elements which can be removed without changing functionality and in this case the expression will be transformed, but not removed. 
3. Remove the `suppressId` attribute as well. The `shortName` will then be used for suppression and it is more correct in my opinion.

Reply:
Changed. Thank you!